tensorflow/tools/ci_build/mac/gpu/pip/run.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ==============================================================================
+#
+# Usage:
```

Comment:
Is it possible to keep the current framework, in which all pip builds go through the ci_parameterized_build and ci_build scripts? For one thing, that will minimize code duplication (e.g., the calling of the configure script, the build_pip_package script, virtualenv etc.) and reduce maintenance cost in the future. 

Is there any reason that I am missing about why it is difficult to stay within that framework? If so, please let me know.